### DISCUSSION TITLE
Copy next/previous item while displaying the item

### DISCUSSION THREAD

**Discussion by Gitoffthelawn** [hluk/CopyQ#1626] [Q&A] [answered]
In https://github.com/hluk/CopyQ/issues/1620#issuecomment-817363391 @sommella wrote:
>I assigned keyboard shortcuts ("copy next item" and "copy previous item") to quickly move between latest clipboards without opening the main windows.
> The notification, configured with a 2 seconds interval, a large geometry (500 width) and a good number of lines (300) helps me choose the right one.

Sounds wonderful.  Can you share how you did this?

**Comment by sommella** [comment, marked as answer]
Hi,
Just configure as the following:

![image](https://user-images.githubusercontent.com/1484873/114512469-6ebed080-9c39-11eb-8c25-42e17f185018.png)

![image](https://user-images.githubusercontent.com/1484873/114512526-80a07380-9c39-11eb-9ec5-948202320218.png)

Regards

**Reply by Gitoffthelawn** [reply]
Thanks so much!  I didn't realize those two commands were pre-configured to show the items as they are rotated into the clipboard.  Cool.
